l10n_eu_adr_report/report/dangerous_good_delivery_report.xml

```diff
@@ -0,0 +1,208 @@
+                        <td><span t-esc="line['volume']"/></td>
+                    </tr>
+                    </t>
+            </tbody>
+        </table>
+    </template>
+
+    <template id="report_delivery_adr_details">
+        <table class="table table-condensed adr-report" >
+            <tbody>
+                <tr>
+                    <td>14 Shipping marks</td>
+                    <td class="wide-col" >Number and kind of packages; description of goods</td>
+                    <td>Gross mass (kg)</td>
+                    <td >Net mass (kg)</td>
+                    <td >Cube (m 3 )</td>
+                </tr>
+                <t t-foreach="first_page_lines" t-as="line">
+                    <tr>
+                        <td/>
+                        <td class="wide-col"><span t-esc="line['class']"/> <span t-esc="line['name']"/> ( <span t-esc="line['un']"/> ) </td>
+                        <td t-esc="line['gross_weight']"></td>
+                        <td><span t-esc="line['weight']"/></td>
+                        <td><span t-esc="line['volume']"/></td>
+                    </tr>
+                    </t>
+                </tbody>
+        </table>
+    </template>
+
+    <template id="report_delivery_adr_document">
+        <t t-call="web.html_container">
+            <t t-call="web.basic_layout">
+                <div class="page adr-report">
+                    <t t-set="o" t-value="o.with_context(lang=o.partner_id.lang)"/>
+                    <h5>
+                        <span class="uppercase emphatic">MULTIMODAL DANGEROUS GOODS FORM</span>
+                    </h5>
+                    <table class="table table-condensed">
+                        <tbody>
+                            <tr>
+                                <td colspan="2" rowspan="3" class="wide-col">1 Shipper/Consignor/Sender</td>
+                                <td colspan="4">2 Transport document number</td>
+                            </tr>
+                            <tr>
+                                <td colspan="2">3 Page of pages</td>
+                                <td colspan="2">4 Shipper’s reference</td>
+                            </tr>
+                            <tr>
+                                <td colspan="2"></td>
+                                <td colspan="2">5 Freight forwarder’s reference</td>
+                            </tr>
+                            <tr>
+                                <td colspan="2">6 Consignee</td>
+                                <td colspan="4">7 Carrier (to be completed by the Carrier)</td>
+                            </tr>
+                            <tr>
+                                <td  colspan="2"></td>
+                                <td class="adr-text" colspan="4"> <span class="uppercase emphatic  adr-text"> SHIPPER’S DECLARATION</span>
+                                    I herby declare that the contents of this consignement are fully and accurately 
```

Comment:
*hereby

Reply:
please, split all these long texts (only the text) to a different sub template.
If the user wants to modify the text only it won't affect the mail template.